_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -21,18 +21,17 @@ This rule applies to any [visible][] HTML element that is a [semantic link][], f
 
 - **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
 - **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
-- **different color**: the element's [foreground color][] and the [foreground color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1, or the element's [background color][] and the [background color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1.
+- **different hue**: the element's [foreground color][] and the [foreground color][] of the **non-link line text** elements have a [different hue][], or the element's [background color][] and the [background color][] of the **non-link line text** elements have a [different hue][].
```

Comment:
Since we are actually comparing sets of colors, shouldn't it be plurals ("foreground colors", …)?